modules/ROOT/pages/download/nb26/index.adoc

```diff
@@ -0,0 +1,139 @@
+:page-tags: download
+:jbake-status: published
+:keywords: Apache NetBeans 25 Download
+:description: Apache NetBeans 25 Download
+:toc: left
+:toc-title:
+:icons: font
+
+// check version above and follow the TODOs
+:netbeans-version: 26
+// TODO            ^^
+
+// https://infra.apache.org/release-distribution.html#download-links
+// TODO base URLs - when archiving, delete the next 2 lines and uncomment the following 2
+:url-download: https://www.apache.org/dyn/closer.lua/netbeans/
+:url-download-keychecksum: https://downloads.apache.org/netbeans/
+// :url-download: https://archive.apache.org/dist/netbeans/
+// :url-download-keychecksum: https://archive.apache.org/dist/netbeans/
+
+//// 
+url-download depends of release status archived or not
+https://archive.apache.org/dist/netbeans/  (//archived)
+https://www.apache.org/dyn/closer.lua/netbeans/ (//current)
+
+url-download-keychecksum depends of release status archived or not
+https://archive.apache.org/dist/netbeans/  (//archived)
+https://downloads.apache.org/netbeans/ (//current)
+////
+
+Apache NetBeans {netbeans-version} was released on May xxx, 2025.
+// TODO update date                                ^^^^
+
+////
+NOTE: It's mandatory to link to the source. It's optional to link to the binaries.
+NOTE: It's mandatory to link against https://www.apache.org for the sums & keys. https is recommended.
+NOTE: It's NOT recommended to link to github.
+////
+
+*Binaries (Platform Independent):*
+
+* link:{url-download}netbeans/{netbeans-version}/netbeans-{netbeans-version}-bin.zip?action=download[netbeans-{netbeans-version}-bin.zip]
+ (link:{url-download-keychecksum}netbeans/{netbeans-version}/netbeans-{netbeans-version}-bin.zip.sha512[SHA-512],
+ link:{url-download-keychecksum}netbeans/{netbeans-version}/netbeans-{netbeans-version}-bin.zip.asc[PGP ASC])
+
+*Source:*
+
+* link:{url-download}netbeans/{netbeans-version}/netbeans-{netbeans-version}-source.zip?action=download[netbeans-{netbeans-version}-source.zip] 
+(link:{url-download-keychecksum}netbeans/{netbeans-version}/netbeans-{netbeans-version}-source.zip.sha512[SHA-512], 
+link:{url-download-keychecksum}netbeans/{netbeans-version}/netbeans-{netbeans-version}-source.zip.asc[PGP ASC])
+
+Officially, it is important that you link:https://www.apache.org/dyn/closer.lua#verify[verify the integrity]
+of the downloaded files using the PGP signatures (.asc file) or a hash (.sha512 files).
+The PGP keys used to sign this release are available link:https://downloads.apache.org/netbeans/KEYS[here].
+
+*Release Notes:*
+
+* link:https://github.com/apache/netbeans/releases/tag/{netbeans-version}[Github Link]
+
+== Community Installers
+
+- link:https://installers.friendsofapachenetbeans.org/[Friends of Apache NetBeans] - Windows, macOS and
+Linux (.deb / .rpm / .AppImage) built with link:https://github.com/apache/netbeans-nbpackage/[NBPackage]. Most
+include a local JDK runtime for the IDE to run on, for a self-contained out-of-the-box experience.
```

Comment:
Remove `.AppImage` - not planned for NB26 at the moment.

Change to "All include a local JDK runtime ..."